src/_scss/pages/state/_contentWrap.scss

```diff
@@ -29,11 +29,20 @@
 
         .state-section__row {
             width: 100%;
-            margin-bottom: rem(50);
+            margin-bottom: rem(32);
 			@include media($tablet-screen) {
 				@include display(flex);
 			}
 
+			&.award-row {
+				margin-bottom: 33px;
+			}
+
+			&.geo-row {
+				padding-bottom: 2px;
```

Comment:
This seems unnecessary, but I may be missing something.
It doesn't seem like this is really doing anything, because there's already a lot of whitespace below this element.
If we need to increase the whitespace, it should be done with margin-bottom